stasis/package.json

```
@@ -0,0 +1,44 @@
+{
+  "name": "@chainlink/stasis",
+  "version": "0.0.1",
+  "description": "Chainlink stasis adapter.",
+  "keywords": [
+    "Chainlink",
+    "LINK",
+    "blockchain",
+    "oracle"
+  ],
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "repository": {
+    "url": "https://github.com/smartcontractkit/external-adapters-js",
+    "type": "git"
+  },
+  "license": "MIT",
+  "scripts": {
+    "prepublishOnly": "yarn build && yarn test:unit",
+    "build": "yarn && tsc",
```

```suggestion
    "build": "tsc",
```
This can drop the `yarn` because it is handled in the `make` command now.